framework/docs/src/5_platform/7_oracle.md

```diff
@@ -0,0 +1,65 @@
+# Account Value Oracle
+
+Have you ever wished to know the value of your portfolio? I'm guessing you have. Luckily Abstract has an integrated system to retrieve the value of your assets, on-chain!
+
+## Value is relative
+
+The value of something is always relative to some other thing. A bitcoin isn't valued at 20,000, it's valued at 20,000 **USD**. Likewise the first decision that's required to work with Abstract's oracle is: In what currency do you want your Account's assets to be valued?
+
+We'll call this currency the *base asset*. There can never be more than one base asset and every asset will be valued in terms of this base asset.
```

Comment:
We should probably mention that this feature is almost always for application developers and not necessarily user accounts?